packages/test/test-service-load/src/workloads/gc/gcDataStores.ts

```diff
@@ -1000,9 +1012,6 @@ export class RootDataObject extends DataObject implements ITestRunner {
 			sendFunc = sendFunc.bind(config.logger);
 			sendFunc(event, logLevel);
 			if (
-				event.eventName.includes("InactiveObject") ||
-				event.eventName.includes("TombstoneReadyObject") ||
-				event.eventName.includes("SweepReadyObject") ||
 				event.eventName.includes("GC_Tombstone") ||
```

Comment:
Since the case we hit will be fixed by Reference Op work, maybe we should leave these?  The only remaining false positive will be for an inactive object that's re-referenced as part of a subtree that's revived.

Eh, I guess that's probably common and may even be in those test failures - the DataStore will be listed in the Reference Op but DDS wouldn't be, I think I remember in the logs there were multiple logs, probably one is for DDS?

So, never mind, it's good to get rid of these :)